cypress/common/base.ts

```diff
@@ -62,17 +62,27 @@ export class BaseMethods {
     public clickElementBySelector({
```

Comment:
Could you add changes to `checkElementState` method above on 43th line and remove duplicate method on 255th line